framework/bootstrap/config.go

```diff
@@ -79,6 +87,24 @@ func patchGlobal(global, patch *bootconfig.Global) {
 		if global.Log.KlogLevel == 0 {
 			global.Log.KlogLevel = patch.Log.KlogLevel
 		}
+		if global.Log.LogRotate {
```

Comment:
consider using lib like `github.com/imdario/mergo` to do these `merge-struct` thing to make the code more brief